### PR TITLE
feat: implement graphql-crunch

### PR DESCRIPTION
**Pull request by xdk78**
related to https://github.com/guildspeak/guildspeak-backend/pull/17

### CODE REVIEW

File: src/renderer/app/client.ts [hunk 1]
```diff
@@ -23,7 +24,10 @@ const authLink = new ApolloLink((operation, forward) => {
     }
   })
   // @ts-ignore
-  return forward(operation)
+  return forward(operation).map((response) => {
+    response.data = uncrunch(response.data)
```
Comment (Bjornskjald) [inline src/renderer/app/client.ts]: Maybe check if data is not undefined?

Comment (Bjornskjald) [reply src/renderer/app/client.ts]: The same in guildspeak/guildspeak-backend#17

Comment (xdk78) [reply src/renderer/app/client.ts]: look at `   if (!operation) throw new Error('operation error')`

Comment (Bjornskjald) [reply src/renderer/app/client.ts]: So... are you sure data won't be undefined?

Comment (xdk78) [reply src/renderer/app/client.ts]: also Prisma automatically throw error if data is not undefined

Comment (Bjornskjald) [reply src/renderer/app/client.ts]: Alright, approving then.

**Pull request merged**